core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -181,6 +185,12 @@ protected BikeCommonFlagEncoder(int speedBits, double speedFactor, int maxTurnCo
         routeMap.put(REGIONAL, VERY_NICE.getValue());
         routeMap.put(LOCAL, PREFER.getValue());
 
+        cyclewayMap.put("lane", UNCHANGED.getValue());
+        cyclewayMap.put("shared_lane", UNCHANGED.getValue());
+        cyclewayMap.put("share_busway", UNCHANGED.getValue());
+        cyclewayMap.put("shoulder", UNCHANGED.getValue());
+        cyclewayMap.put("track", PREFER.getValue());
+
```

Comment:
This way to write priorities should make it a lot easier for us to tweak these values. I don't agree with these values as written, but these are what the master branch has. We can think about what the priorities should be in a later PR.